test/e2e/load/load.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2021 ScyllaDB
+
+package load
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx/v2"
+	"github.com/scylladb/gocqlx/v2/table"
+)
+
+type Load struct {
+	session  gocqlx.Session
+	table    *table.Table
+	expected []*test
+}
+
+type test struct {
+	Id   int
+	Data string
+}
+
+func NewLoad(hosts ...string) (*Load, error) {
+	session, err := newSession(hosts...)
+	if err != nil {
+		return nil, err
+	}
+
+	l := &Load{
+		session: session,
+		table: table.New(table.Metadata{
+			Name:    "test.test",
+			Columns: []string{"id", "data"},
+			PartKey: []string{"id"},
+		}),
+		expected: nil,
```

Comment:
would it be safer to initialize expected data here so whenever you compare a result of a database list, you don't have to think about other states?

Reply:
ditto